core/vm/jump_table.go

```diff
@@ -1059,12 +1073,17 @@ func newFrontierInstructionSet() JumpTable {
 			minStack:   minStack(1, 0),
 			maxStack:   maxStack(1, 0),
 		},
+		INVALID: {
+			execute:  opUndefined,
+			minStack: minStack(0, 0),
+			maxStack: maxStack(0, 0),
+		},
```

Comment:
This is IMO very wonky, see discussion here: https://github.com/ethereum/go-ethereum/pull/29518/files#r1753395744 .